File: solutions/2752-sum-multiples/solution.py
class Solution(object):
    def sumOfMultiples(self, n):
        """
        :type n: int
        :rtype: int
        """
        m=0
        for i in range(1,n+1):
            if i%3==0 or i%5==0 or i%7==0:
                m+=i
        return m

File: solutions/2752-sum-multiples/solution.js
/**
 * @param {number} n
 * @return {number}
 */
var sumOfMultiples = function(n) {
    var m=0;
        for(let i=0;i<=n;i++)
            if(i%3==0 || i%5==0 || i%7==0)
                m+=i;
        return m;
};
